File: app/src/main/java/com/kkkkkn/readbooks/view/view/BrowsingActivityView.java
package com.kkkkkn.readbooks.view.view;

import com.kkkkkn.readbooks.model.entity.ChapterInfo;

import org.json.JSONArray;

import java.util.ArrayList;

public interface BrowsingActivityView extends BaseView {
    void syncChapterList(ArrayList<ChapterInfo> list);
    void toLoginActivity();
    void syncReadView(JSONArray jsonArray);
    void setLoading(boolean type);
}

File: app/src/main/java/com/kkkkkn/readbooks/view/view/BrowsingActivityView.kt
package com.kkkkkn.readbooks.view.view

import com.kkkkkn.readbooks.model.entity.ChapterInfo
import org.json.JSONArray

interface BrowsingActivityView :BaseView {
    fun syncChapterList(list: ArrayList<ChapterInfo>)
    fun toLoginActivity()
    fun syncReadView(jsonArray: JSONArray)
    fun setLoading(type: Boolean)
}